contracts/payloads/IGP95/PayloadIGP95.sol

```diff
@@ -323,7 +281,7 @@ contract PayloadIGP95 is PayloadIGPMain {
         }
     }
 
-    // @notice Action 7: Pause Limits for Unlaunched USDe Collateral Vaults
+    // @notice Action 7: Pause Limits for Bugged DEXes
```

Comment:
let's not call this bugged dexes but rather "replaced"... bugged is a strong word :smile: